plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveAnalyzeCorruptStatistics.java

```diff
@@ -78,6 +78,37 @@ private void prepareBrokenColumnStatisticsTable(String tableName)
         assertEquals(onMetastore("SELECT COUNT(1) FROM TAB_COL_STATS WHERE db_name = 'tpch' AND table_name = '" + tableName + "'"), "2");
     }
 
+    @Test
+    public void testAnalyzeCorruptPartitionStatisticsOnEmptyTable()
+    {
+        String tableName = "test_analyze_corrupt_partition_statistics_" + randomNameSuffix();
+
+        // Concurrent ANALYZE statements generate duplicated rows in Thrift metastore's PART_COL_STATS table when partition statistics is empty
+        prepareBrokenPartitionStatisticsTable(tableName);
+
+        // SHOW STATS should succeed even when the partition statistics is broken
+        assertQuerySucceeds("SHOW STATS FOR " + tableName);
+
+        // ANALYZE succeeds unlike TAB_COL_STATS, but drop_stats fails for tables having broken partition statistics
```

Comment:
> ANALYZE succeeds unlike TAB_COL_STATS

What do you mean here?

Did you imply ?

> ANALYZE succeeds regardless the TAB_COL_STATS table being corrupt

Reply:
No. There's another test case in this class `testAnalyzeCorruptColumnStatisticsOnEmptyTable` that uses `TAB_COL_STATS`. ANALYZE statement throws an exception if `TAB_COL_STATS` contains duplicated rows.